docker-compose.yml

```diff
@@ -54,16 +54,8 @@ services:
     build:
       context: docker/images/fakesentry
     image: local/eliot_fakesentry
-    ports:
-      - "8090:8090"
```

Comment:
We need this to connect to fakesentry to view errors.

I'm fine with putting it on a different port, but it needs to be exposed.

Reply:
I'll make it configurable with env vars, and default to the existing port